### DISCUSSION TITLE
Optional Config - EMAIL

### DISCUSSION THREAD

**Discussion by terafirmanz** [nautobot/nautobot#4022] [General]
I noticed that the documentation and even example config files do not list any EMAIL config options to setup an SMTP server. Is this still supported and is it done the same as Netbox or does it have a new config syntax?

**Comment by glennmatthews** [comment]
There's nothing specific to Nautobot for email configuration - the [Django documentation](https://docs.djangoproject.com/en/3.2/ref/settings/#email-backend) would apply here. A pull request to add an entry to Nautobot's configuration docs pointing there would probably be welcome though.
